api/reporting/schemas/latest/user.json

```diff
@@ -0,0 +1,175 @@
+[
+  {
+    "description": "Time snapshot was taken. Same across all rows in the snapshot, and can be used for partitioning.",
+    "name": "snapshot_timestamp",
+    "type": "INTEGER"
+  },
+  {
+    "name": "about_you",
+    "type": "STRING"
+  },
+  {
+    "name": "area_of_research",
+    "type": "STRING"
+  },
+  {
+    "name": "beta_access_bypass_time",
+    "type": "TIMESTAMP"
```

Comment:
BQ timestamps don't have zones; it's always assumed to be UTC (or whatever timezone you want if you're consistent).